docs/book/src/developer/providers/migrations/v1.5-to-v1.6.md

```diff
@@ -34,7 +34,7 @@ maintainers of providers and consumers of our Go API.
 ### Other
 - `clusterctl move` can be blocked temporarily by a provider when an object to be moved is annotated with `clusterctl.cluster.x-k8s.io/block-move`.
 - `mdbook releaselink` has been changed to require a `repo` tag when used in markdown files for generating a book with `mdbook`.
-
+- Workload cluster Node yaml is now being collected during e2e tests to assist with troubleshooting test failures. 
```

Comment:
Can we mention that `DumpKubeSystemPodsForCluster` got renamed to `DumpResourcesForCluster` and what resources it now dumps in addition to kube-system pods?